### DISCUSSION TITLE
Welcome to luckfox-pico Discussions!

### DISCUSSION THREAD

**Discussion by luckfox-eng33** [LuckfoxTECH/luckfox-pico#119] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by plan44** [comment]
HI everyone, I'm a self employed maker of smarthome devices based on small Linux capable SoCs. So I'm excited about the Luckfox Pico, because it is one of only few Ethernet enabled devices with a much smaller physical and power footprint than the Raspberry class of devices (which are way too big for my purposes).

I'm primarily looking to run OpenWrt on the Pico Max, and if possible, would like to help with this effort that @pengphei [has already started](https://github.com/LuckfoxTECH/luckfox-pico/issues/115). I would also want to control WS28xx LEDs with the pico (see #135) and might be able to contribute to that.

What I am looking for at the moment is a more complete datasheet / programmers manual for the RV1106, in particular for the IR capable PWM units.
